appendix-basic65.tex

```diff
@@ -4014,7 +4025,7 @@ \subsection{FRE}
                {\bf FRE(1)} returns the number of free bytes in
                bank 1, which is the bank for BASIC variables, arrays
                and strings. {\bf FRE(1)} also triggers
-               "garbage collection", which is a process that collects
+               ``garbage collection'', which is a process that collects
                used strings at the top of the bank, thereby
```

Comment:
"strings in use" describes the status better, than "used strings"

Reply:
fixed